build.cmd

```diff
@@ -432,6 +433,7 @@ if not exist "!IbcDataPackagePathOutputFile!" (
 )
 
 set /p __IbcOptDataPath=<"!IbcDataPackagePathOutputFile!"
+set __IbcOptDataPath="%__IbcOptDataPath%"
```

Comment:
It is bad practice in batch files to quote paths in environment variables. Instead, quote the entire value at usage.